stylis.js

```diff
@@ -1250,6 +1250,22 @@
 
 				break
 			}
+			// margin-inline-start, margin-inline-end: m, a, r
+			case 965: {
```

Comment:
Since the logic within the block is the same for padding and margin:

```diff
+ case 915: case 965: {
```

and test for if indexOf > 5.